BUMO/week-3-2/src/chat/menu/template.pug

```diff
@@ -1,8 +1,6 @@
 h1 BUMO
 
-//button(ng-click="popoverVisible=!popoverVisible" outer-click="popoverVisible=false" outer-click-exclude="popover") Threads
-    span(ng-if="$ctrl.count") &nbsp;({{ $ctrl.count }})
-button(ng-click="popoverVisible=!popoverVisible" outer-click="popoverVisible=false" outer-click-exclude="popover") Shuffle!
-    span(ng-if="$ctrl.count") &nbsp;({{ $ctrl.count }})
+button(ng-click="$ctrl.setItem('Shuffle!')") Shuffle!
```

Comment:
the other button does set a `Shuffle` and this one sets `Shuffle!` is it normal?